src/setup-gcloud.ts

```diff
@@ -55,12 +55,20 @@ export async function run(): Promise<void> {
     }
 
     const serviceAccountKey = core.getInput('service_account_key');
-    // If a service account key isn't provided, log an un-authenticated notice
-    if (!serviceAccountKey) {
-      core.info('No credentials provided, skipping authentication');
-      return;
-    } else {
+    // If a service account key is provided, add warning to use google-github-actions/auth
+    if (serviceAccountKey) {
+      core.warning(
```

Comment:
TBD if we want to add this in setup-gcloud. I kept it for consistency with the refactors we did in other actions.